src/pickers/TaskExecutorsPicker.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import {
+  useTranslations, Autocomplete, useModulesManager, useGraphqlQuery,
+} from '@openimis/fe-core';
+
+function TaskExecutorsPicker({
+  multiple = true,
+  withLabel,
+  onChange,
+  value,
+  readOnly,
+  required,
+}) {
+  const [searchString, setSearchString] = useState();
+  const modulesManager = useModulesManager();
+  const { formatMessage } = useTranslations('tasksManagement', modulesManager);
+
+  const { isLoading, data, error } = useGraphqlQuery(
+    `
+    query TaskExecutorsPicker($str: String) {
+      users(str: $str) {
+        edges {
+          node {
+            id
+            username
+            lastName
+          }
+        }
+      }
+    }
+  `,
+    { str: searchString },
+  );
+
+  const uniqueValues = [...new Map(value?.map((user) => [user.id, user])).values()];
+
+  const executors = data?.users?.edges.map((edge) => edge.node) ?? [];
+
+  const filterOptions = (users, { inputValue }) => {
+    const filteredUsers = users?.filter((user) => {
+      const username = user.username.toLowerCase();
+      const lastName = user.lastName.toLowerCase();
+      const input = inputValue.toLowerCase();
+
+      return username.includes(input) || lastName.includes(input);
+    });
+    return filteredUsers.filter((user, i, arr) => arr.findIndex((u) => u.id === user.id) === i);
+  };
```

Comment:
I think that this should be resolved on the graphql level. If logic is not compliant with the users query then we can create custom search query for task triage candidate purposes

Reply:
As I mentioned before, I didn't want to solve this via graphql filters. Let me know if this approach is acceptable for you.

Reply:
As long as we pull all entries in one query it's fine but I don't think that solution is sustainable in the long term. Please create ticket in the OP board to address dropdowns that pull everything.